tools/build.js

```diff
@@ -118,11 +117,11 @@ var PROJECT;
 // Short-form of PROJECT.version
 var VERSION;
 
-// Root POM tasks
-var TASKS;
+// Root POM tasks and exports
+var TASKS, EXPORTS;
 
 // These are different for an unknown historic reason and should be merged.
-var BUILD_DIR  = './build2', TARGET_DIR = './build2';
+var BUILD_DIR  = './build2', TARGET_DIR = './target';
```

Comment:
Why do this?

Reply:
Before genjava was meant to generate only java source files from foam models, dependencies resolution, compilation and jar packaging was done in gradle and bundling resources, documents and journals was done in various bash scripts.

Now genjava can do more but there is still a good reason I guess to have the other resources in the target directory as to separate the java vs. non-java resources for modern java IDE like intelliJ to have simple rule to configure source directory for coding assistance functionality while ignoring the others.